activities/staff-management/onboarding.md

```diff
@@ -30,9 +30,9 @@ The Operations Coordinator needs to add the new employee to Tentoo NMBRS:
 If the employee agrees to the terms and has no further questions, onboarding takes place as follows:
 
 * Contract is converted from draft to final version and uploaded to Docusign
-* The Chief Executive, President of the board and the employee need to sign
-* Once everyone has signed the contract, the fully signed version is sent to the employee by the Operations Coordinator
-* In the same email, the Operations Coordinator also sends a welcome message and our onboarding package
+* The chief executive, president of the board of directors and the employee need to sign
```

Comment:
The president role is not a board role. All board roles are volunteer roles. The president role is expressly not a board role since it is not a volunteer role. Good that we cought it here!

```suggestion
* The chief executive, president and the employee need to sign
```

Reply:
From skimming the Articles of Association, it looks like the chair is the board role, while the president is actually part of the secretariat team. Can I confirm that the intention in this hiring process was for the president to be involved, not the chair of the board?

Reply:
yes, the president co-signs with the chief executive

Reply:
For a contract 2 board signatures are necessary. That is usually the two board of directors members most involved with the day to day, for now, Ben and me, although other board members have signed contracts in the past.

**It is not the President that signs**, since that role does not have signing authority as is not a board role because it is paid. It is critical we keep these roles separate in line with the remuneration policy.

I understand it is confusing that the role of chairman of the board is perfomed by Ben and president is contracted out to Multiple Ventures BV and executed by him as well, however these are separate roles with separate responsibilities and have to be treated as such.

Reply:
I wish I had seen this sooner as well!

Reply:
In addition, @ElenaFdR, that I have both the role of Secretary and am chief executive does not make our staff a secretariat team, we have not defined anywhere that the staff explicitly fall under the secretary, but under the board of directors as a whole. Again, this is important to separate the voluntary board duties and the paid staff/contractor duties.